README.md

```diff
@@ -1,6 +1,5 @@
 # EVE is Edge Virtualization Engine
 
-[![CircleCI](https://circleci.com/gh/lf-edge/eve.svg?style=svg)](https://circleci.com/gh/lf-edge/eve/tree/master)
```

Comment:
Instead of just deleting this -- can you please provide an equivalent URL for a GitHub actions build process?

Reply:
We will add a new one from Github Actions after fixing the Build PR workflow.

Reply:
Added GH action one. Publish workflow indicator